ext/ffi/lib.rs

```diff
@@ -108,6 +112,9 @@ pub fn init<P: FfiPermissions + 'static>(unstable: bool) -> Extension {
       ("op_ffi_load", op_sync(op_ffi_load::<P>)),
       ("op_ffi_call", op_sync(op_ffi_call)),
       ("op_ffi_call_nonblocking", op_async(op_ffi_call_nonblocking)),
+      ("op_ffi_ptr_of", op_sync(op_ffi_ptr_of)),
```

Comment:
Unsafe Pointer class' methods like static `of`, `read`, `readCString` should require `allow-ffi` permission IMO

Reply:
Hmm, how would I implement this? `FfiPermissions::check` requires a `path` component, should i expose and use `FfiDescriptor::query` or how would i check if `allow-ffi` is enabled without checking any specific path?